src/WebJobs.Script/Settings/ISettingsManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Settings
+{
+    public interface ISettingsManager
```

Comment:
Recommend removing this interface and just having the ScriptSettingsManager class. The various methods needed for mock testing can be made virtual as needed.

The reason I think this will be better is because I'd like to start adding first class properties to the manager for accessing common properties (e.g. WebAppName) and I'd rather not have to add that to both an interface and a class in each case. All our scenarios can be accomplished with just a class I think. Thoughts?